app/lib/services/funding_eligibility.rb

```diff
@@ -74,10 +91,19 @@ def ineligible_institution_type?
 
   private
 
+    def lead_mentor_eligibility_status
+      if lead_mentor_course?
+        FUNDED_ELIGIBILITY_RESULT
```

Comment:
Oh I was just about to approve but noticed an issue. If you have an approved ITT Provider and are a lead mentor, and pick the right course, you can be funded multiple times. We just need to add the previously funded check in here to make sure we still check for previous funding:

```suggestion
        return PREVIOUSLY_FUNDED if previously_funded? 
        
        FUNDED_ELIGIBILITY_RESULT
```

With that in place we'll be good to go